crates/sncast/src/main.rs

```diff
@@ -402,6 +400,36 @@ async fn run_async_command(
     }
 }
 
+fn run_script_command(
+    cli: &Cli,
+    runtime: Runtime,
+    config: &CastConfig,
+    script: &Script,
+    numbers_format: NumbersFormat,
+    output_format: &OutputFormat,
+) -> Result<()> {
+    if let Some(starknet_commands::script::Commands::Init(init)) = &script.command {
+        let mut result = starknet_commands::script::init::init(init);
+        print_command_result("script init", &mut result, numbers_format, output_format)?;
+    } else {
+        let provider = get_provider(&config.rpc_url)?;
+        let script_module_name = script.script_module_name.as_ref().ok_or_else(|| {
+            anyhow!("required positional argument SCRIPT_MODULE_NAME not provided")
```

Comment:
This line is a temporary solution and it will be removed in https://github.com/foundry-rs/starknet-foundry/pull/1320